test/BlockBuilderPolicy.t.sol

```diff
@@ -37,7 +32,8 @@ contract BlockBuilderPolicyTest is Test {
         ),
         publicKey: hex"bf42a348f49c9f8ab2ef750ddaffd294c45d8adf947e4d1a72158dcdbd6997c2ca7decaa1ad42648efebdfefe79cbc1b63eb2499fe2374648162fd8f5245f446",
         teeAddress: 0xf200f222043C5bC6c70AA6e35f5C5FDe079F3a03,
-        workloadId: WorkloadId.wrap(0xeee0d5f864e6d46d6da790c7d60baac5c8478eb89e86667336d3f17655e9164e)
+        workloadId: WorkloadId.wrap(0xeee0d5f864e6d46d6da790c7d60baac5c8478eb89e86667336d3f17655e9164e),
+        privateKey: 0x0000000000000000000000000000000000000000000000000000000000000000 // unused for this mock
```

Comment:
I spent some time trying to refactor `MockQuote` into its own library/contract, but it ended up being more work than it's worth, so I abandoned it.